### PR TITLE
Feature/doc 209 deployment script fix

### PR DESCRIPTION
**Pull request by max402**

### CODE REVIEW

File: .travis/deploy_dockerhub.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```
Comment (valb3r) [inline .travis/deploy_dockerhub.sh]: Can't see how this file is used

Comment (max402) [reply .travis/deploy_dockerhub.sh]: now it's used

File: datasafe-rest-impl/datasafe-dockerhub-image.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+kind: ImageStream
+apiVersion: v1
+metadata:
+  name: datasafe
+spec:
+  tags:
+    - from:
+        kind: DockerImage
+        name: docker.io/max402/datasafe:latest
```
Comment (valb3r) [inline datasafe-rest-impl/datasafe-dockerhub-image.yaml]: Shouldn't we deploy to adorsys account?

Comment (max402) [reply datasafe-rest-impl/datasafe-dockerhub-image.yaml]: forgot to delete this file. It's unused

**Pull request merged**